src/test_typing.py

```diff
@@ -20,6 +20,25 @@
 import typing
 
 
+class TestCase(_TestCase):
+
+    # vanilla TestCase doesn't have assert*Subclass, see bugs.python.org/14819
```

Comment:
Right, the comment was not meant to imply that `assertIsSubclass` would eventually be a standard feature.  The rename dance was an effort to avoid churn, but as you pointed out, the whole PR is pretty much churn so I'll just change it to BaseTestCase.